app/scripts/templates/settings/account_recovery/confirm_password.mustache

```diff
@@ -0,0 +1,24 @@
+<div id="account-recovery-confirm-password">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Re-enter password{{/t}}</h2>
+
+        <form novalidate>
+            <p class="description">
+                {{#t}}Enter your password to view your recovery key.{{/t}}
+            </p>
+
+            <h2 class="email">{{email}}</h2>
```

Comment:
If the styling for this element is retained, can you change this from an h2 since this isn't really a header?